topics/imaging/tutorials/voronoi-segmentation/tutorial.md

```diff
@@ -0,0 +1,312 @@
+Therefore, we first smooth the image to reduce the influence of noise, and then apply a threshold on the smoothed image to get a binary mask. 
+> <comment-title> Mask vs seeds </comment-title>
+> The process used to create a mask can also be used to make seeds, as in the 
+> [Imaging introduction tutorial](https://training.galaxyproject.org/training-material/topics/imaging/tutorials/imaging-introduction/tutorial.html). 
+{: .comment}
+
+> <hands-on-title> Task description </hands-on-title>
+> The image has three channels (red, green, blue). To generate a mask, we have to select a channel, for instance channel `0`. 
+> 1. {% tool [Convert image format with Bioformats](toolshed.g2.bx.psu.edu/repos/imgteam/bfconvert/ip_convertimage/6.7.0+galaxy3) %} with the following parameters:
+>    - {% icon param-file %} *"Input Image"*: `image`
+>    - *"Extract series"*: `All series`
+>    - *"Extract timepoint"*: `All timepoints`
+>    - *"Extract channel"*: `Extract channel`
+>        - *"Channel id"* `0`
+>    - *"Extract z-slice"*: `All z-slices`
+>    - *"Extract range"*: `All images`
+>    - *"Extract crop"*: `Full image`
+>    - *"Tile image"*: `No tiling`
+>    - *"Pyramid image"*: `No Pyramid`
+> 
+>    <!-- - **Important:** If using the tree image, choose the type of data as `zip`. --> 
+> 
+> 2. Rename the output to `single channel image`
+> 
+> 
+> 3. {% tool [Filter 2-D image with Scipy](toolshed.g2.bx.psu.edu/repos/imgteam/2d_simple_filter/ip_filter_standard/1.12.0+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Input Image"*: `single channel image`
+>    - *"Filter type"*: `Gaussian`
+>        - *"Sigma"*: `3`
+> 
+> 4. Rename the output to `smoothed image`.
+> 
+>
+> 5. {% tool [Threshold image](toolshed.g2.bx.psu.edu/repos/imgteam/2d_auto_threshold/ip_threshold/0.18.1+galaxy3) %} with the following parameters:
+>    - {% icon param-file %} *"Input Image"*: `smoothed image`
+>    - *"Thresholding method"*: `Manual`
+>        - *"Threshold value"*: `3.0`
+> 
+> 6. Rename the output to `mask`.
+> 
+{: .hands_on}
+
+> <comment-title> How many channels does my image have? </comment-title>
+> Note: If providing your own image, you can check how many channels your image has with the tool `Show image info with Bioformats`. 
+> The number of channels is listed as e.g. `SizeC = 3` for the HeLa cell image or `SizeC = 3 (effectively 1)` for the tree image.
+{: .comment}
+
+> <comment-title> The value of Sigma and the Threshold value </comment-title>
+> Note: Generating a robust mask is harder for images with more noise. 
+> Since the tree image has more noise than the cell image, you may have to adjust the Sigma to achieve better results. 
+> You may also have to adjust the Threshold value in the last step. 
+{: .comment}
+
+
+> <question-title></question-title>
+>
+> 1. What is the purpose of the smoothing step? 
+>
+> > <solution-title></solution-title>
+> >
+> > 1. The purpose of smoothing is to reduce noise. 
```

Comment:
There is no point in using numbers here, because there is only one item:
```suggestion
> What is the purpose of the smoothing step? 
>
> > <solution-title></solution-title>
> >
> > The purpose of smoothing is to reduce noise. 
```